pkg/metrics/collector_test.go

```diff
@@ -467,7 +482,28 @@ var _ = Describe("ResourcesMetricsCollector", func() {
 				To(Succeed())
 		})
 
-		It("should produce correct metrics from target namespaces", func() {
+		It("should produce correct metrics with cluster scope - Info", func() {
+			const expected = `
+        # HELP trivy_resource_configaudits Number of failing resource configuration auditing checks
+        # TYPE trivy_resource_configaudits gauge
+        trivy_resource_configaudits{name="configmap-test",namespace="some-ns",resource_kind="ConfigMap",resource_name="test",config_audit_id="car2 Id",config_audit_title="car2 config audit title", config_audit_description="car2 description for config audit", config_audit_category="car2 category for config audit",config_audit_success="true",severity="Critical"} 0
```

Comment:
it doesn't make sense metric should name  `trivy_configaudits_info` but I see it has summary metric name `trivy_resource_configaudits`

Reply:
I will take a look at it

Reply:
@chen-keinan what do you suggest?